### PR TITLE
Do not show location in device overview if location is not found

### PR DESCRIPTION
**Pull request by murrant**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/html/dev-overview-data.inc.php [hunk 1]
```diff
@@ -121,11 +121,9 @@
     echo "<div class='row'><div class='col-sm-4'>$uptime_text</div><div class='col-sm-8'>$uptime</div></div>";
 }
 
-if ($device['location_id']) {
+if ($device['location_id'] && $location = Location::find($device['location_id'])) {
```
Comment (SourceDoctor) [inline includes/html/dev-overview-data.inc.php:124]: isn't there a reduction possible to:
```
$location = Location::find($device['location_id']);
if ($location) {
```

Comment (murrant) [reply includes/html/dev-overview-data.inc.php:124]: @SourceDoctor That causes an extra SQL query in some cases.

Comment (SourceDoctor) [reply includes/html/dev-overview-data.inc.php:124]: ok :-)

**Review by SourceDoctor — APPROVED** [review]
LGTM

**Pull request merged**